### ISSUE TITLE
feature: disable (old) transport-in-placement logic

### ISSUE THREAD

**Issue by vMaroon** [kubestellar/kubestellar#1730]
### Feature Description

After #1681 was merged, now is the time to disable all old transport object-wrapping paths and the logic around it.
As a side-job in this task, I will also clean up unrelated, unused code.

This updates the placement controller's responsibilities to the single one of maintaining PlacementDecision (soon Binding) objects.

This task is expected to delete 600-700 lines of code.

### Proposed Solution

Update code

### Want to contribute?

- [ ] I would like to work on this issue.

**Comment by vMaroon** [comment]
this is part of #1520

**Comment by MikeSpreitzer** [comment]
Deleting the old transport code has to wait until the new transport code is fully engaged.

**Comment by vMaroon** [comment]
@MikeSpreitzer I am aware of that, note that the PR I opened deletes currently unused code. That is functions and constants with 0 usage, probably forgotten during the initial development.

In the next PR which will be on hold until transport is engaged, I will delete previously used but then redundant code.
